backend/api/oauthApi.test.ts

```diff
@@ -5,7 +5,7 @@ const clientId = 'clientId'
 const url = 'http://localhost/'
 const clientSecret = 'clientSecret'
 
-const client = {}
+const client = {} as any
```

Comment:
This isn't great but I suppose it's better than not having a type... does the client not actually have an object?